shared/params.go

```diff
@@ -553,7 +553,11 @@ func ParseBooleanParam(r *http.Request, name string) (result *bool, err error) {
 }
 
 func ParseRunIDsParam(r *http.Request) (ids []int64, err error) {
-	idStrs := strings.Split(r.URL.Query().Get("run_ids"), ",")
+	str := r.URL.Query().Get("run_ids")
+	if str == "" {
+		return nil, nil
```

Comment:
@Hexcles should we use `nil` slice or the already established `ErrMissing` here?

Reply:
If `run_ids` is not required, then let's return nil and no error if run_ids is empty. I think ErrMissing should be reserved for missing mandatory fields.